### PR TITLE
Altcoin updates

### PR DESCRIPTION
**Pull request by DarrenWestwood**
altcoin code updates from WordPress #54
Updated to use angularjs for checkout pages

### CODE REVIEW

File: js/app.js [hunk 1]
```diff
@@ -0,0 +1,502 @@
...
+});
+
+service.factory('AltcoinLimits', function($resource) {
+    var rsc = $resource(flyp_base + '/order/limits/:coin/BTC', {
+        coin: '@coin'
+    });
+    return rsc;
+});
+
+service.factory('WpAjax', function($resource) {
```
Comment (jusasiiv) [inline js/app.js]: Refactor the service name. `WpAjax` will be confusing later, this is basically an internal API call so maybe something referring to that

**Comment by jusasiiv** [comment]
Minor issue on refund screen, escape char is not working properly:
`If you don\'t get refunded in a few hours`
Screenshot_2019-10-17 Bitcoin Payment - Company Name

File: js/app.js (hunk 1 above)
Comment (DarrenWestwood) [reply js/app.js]: Refactor in 49cf590

**Comment by DarrenWestwood** [comment]
> Minor issue on refund screen, escape char is not working properly

Fixed in 1eed73b

**Comment by DarrenWestwood** [comment]
> There are two qr code libraries, is it necessary? `js/qrcode-generator/qrcode.js` and `js/angular-qrcode/angular-qrcode.js `

I am currently receiving the following errors when only including the file from WordPress
`ReferenceError: qrcode is not defined`
`ReferenceError: qrcode is not a function`

Using all 3 files from https://github.com/monospaced/angular-qrcode resolved this.

**Comment by jusasiiv** [comment]
> > There are two qr code libraries, is it necessary? `js/qrcode-generator/qrcode.js` and `js/angular-qrcode/angular-qrcode.js `
> 
> I am currently receiving the following errors when only including the file from WordPress
> `ReferenceError: qrcode is not defined`
> `ReferenceError: qrcode is not a function`
> 
> Using all 3 files from https://github.com/monospaced/angular-qrcode resolved this.

I see, they are all part of the same library. Just looked like they were separate libraries

**Comment by blockonomics** [comment]
Screenshot from 2019-10-18 17-19-44
Please fix above

**Pull request merged**